app/build.gradle

```diff
@@ -422,75 +342,6 @@ clean.doLast {
 }
 
 // -------------------------------------------------------------------------------------------------
-// L10N: Verify locales
-// -------------------------------------------------------------------------------------------------
-
-// Fetches the available locales - this doesn't always match the actually available locales (getEnabledLocales()),
-// e.g. if there were import problems:
-def getTranslatedLocales() {
-    def localesDir = file('../l10n-repo/locales')
-
-    def potentialLanguageDirs = localesDir.listFiles(new FilenameFilter() {
-        @Override
-        boolean accept(File dir, String name) {
-            return !name.equals("templates");
-        }
-    })
-
-    def langs = potentialLanguageDirs.collect {
-        // Reduce from list of files to actual names
-        return it.name
-    } .collect {
-        switch(it) {
-            case "id":
-                return "in"
-            case "he":
-                return "iw"
-            default:
-                return it;
-        }
-    } .collect {
-        return '"' + it + '"'
-    }
-
-    langs << "\"en-US\""
-
-    return langs
-}
-
-task verifyLocales << {
```

Comment:
I didn't see this ported over, but is it because it's included (warnings at least) in ImportCommand?

Reply:
Yeah, and going forward we explicitly do not want to import all locales in all cases. :)